pyquil/device.py

```diff
@@ -226,10 +260,11 @@ def fBellStates(self):
         """
         Get a dictionary of two-qubit Bell state fidelities (normalized to unity)
         from the specs, keyed by targets (qubit-qubit pairs).
-
         :return: A dictionary of Bell state fidelities, normalized to unity.
         :rtype: Dict[tuple(int, int), float]
         """
+        warnings.warn(DeprecationWarning("fBellState device specs will be deprecated in a coming "
```

Comment:
lets be more specific -- how about release v2.13

Reply:
also lets change the message to "fBellState device specs have been deprecated, and will be removed in release v2.13 (targeted for October 2019)"